DCP UI (#2747)
SkylineCommunications/dataminer-docs: release-notes/Cloud/DCP_change_log.md

#### 13 February 2024 - Fix - Catalog versions displayed in wrong order [ID_38762]

> The versions of a catalog item will now be sorted correctly.

Reviewer: ```suggestion
The versions of a catalog item will now be sorted by using natural sorting instead of default string comparison.
```

Author: to technical for users imo, not gonna apply

While remote accessing web apps via dataminer.services (e.g. the Monitoring app), the following message could appear: `An unknown error occurred (status: 200).` The app would also stop working until the page was refreshed. This issue has been resolved.

#### 12 January 2024 - Fix - The given username was not applied when deploying a DaaS Express system

> When a custom username was given when deploying a new DaaS Express system the default username was still being used. This has now been resolved.

Reviewer: ```suggestion
When a custom username was given when deploying a new DaaS system the default username (Administrator) was still being used. The user will now be created with the custom username.
```

You can manually delete a DaaS system from dataminer.services, just like any other DMS; however, note that this is irreversible, and all data of the system will be lost.

#### 28 November 2023 - Enhancement - Deploying multiple DaaS Express systems simultaniously

> From now on it is possible to deploy more than one DaaS Express system simultaniously on [dataminer.services](https://dataminer.services).

Reviewer: ```suggestion
From now on it is possible to create more than one DaaS Express system simultaneously on [dataminer.services](https://dataminer.services).
```

#### 24 November 2023 - Fix - Improved username validation when deploying a DaaS Express system

The username validation has been improved to prevent the deployment from failing when the given username was not a valid. This issue was introduced on 10 November 2023.

#### 24 November 2023 - Enhancement - Admin app improved login system

From now on the [Admin app](https://admin.dataminer.services) has an improved login system that should prevent login redirect loops that could be experienced before.

#### 24 November 2023 - New feature - Introducing the new Catalog user interface

> From now on [catalog.dataminer.services](https://catalog.dataminer.services) will have a completely renewed user interface.

Reviewer: ```suggestion
This update includes a new UI for [catalog.dataminer.services](https://catalog.dataminer.services).
```

#### 22 November 2023 - Enhancement - dataminer.services homepage improved login system

From now on [dataminer.services](https://dataminer.services) has an improved login system that should prevent login redirect loops that could be experienced before.

#### 21 November 2023 - New feature - Remote access using DataMiner Cube [ID_37841]

It is now possible to connect to a DataMiner System via remote access using DataMiner Cube. If remote access is enabled for a DMS and you have been granted access to dataminer.services features, you can access the DMS remotely via Cube using the same URL as for remote access to the web pages, but without the protocol prefix `https://`. A button is also available on dataminer.services and in the Admin app that can be used to open Cube with the correct remote access filled in as the host.

> [!NOTE]
> At present, there is still a limitation to this feature: if the DMS has SAML authentication configured, users will not be able to access the DMS remotely with Cube.

#### 10 November 2023 - Fix - Grant admin consent linking button unresponsive

> The grant admin consent linking button wouldn't work anymore. This issue has now been resolved.

Reviewer: ```suggestion
Users were no longer able to grant admin consent for the Teams Chat Integration because the button was no longer functional. This issue has now been resolved.
```

#### 10 November 2023 - Enhancement - Provide a custom username and password when deploying a DaaS Express system

> From now on you will be able to provide your own username and password for your admin account when deployen a DaaS Express system.

Reviewer: ```suggestion
From now on you will be able to provide your own username and password for your admin account when deploying a DaaS system.
```
